src/main/java/net/kaczmarzyk/spring/data/jpa/web/WebRequestProcessingContext.java

```diff
@@ -65,11 +62,11 @@ public QueryContext queryContext() {
 
 	public String getPathVariableValue(String pathVariableName) {
 		if(resolvedPathVariables == null) {
-			resolvedPathVariables = PathVariableResolver.resolvePathVariables(pathPattern(), actualWebPath());
+			resolvedPathVariables = (Map) webRequest.getAttribute(HandlerMapping.URI_TEMPLATE_VARIABLES_ATTRIBUTE, WebRequest.SCOPE_REQUEST);
 		}
-		String value = resolvedPathVariables.get(pathVariableName);
-		if (value != null) {
-			return value;
+
+		if (resolvedPathVariables != null && resolvedPathVariables.get(pathVariableName) != null) {
+			return resolvedPathVariables.get(pathVariableName);
 		} else {
```

Comment:
It seems that NPE could be thrown in line 68. From `HandlerMapping.URI_TEMPLATE_VARIABLES_ATTRIBUTE` javadoc:
```
	/**
	 * Name of the {@link HttpServletRequest} attribute that contains the URI
	 * templates map, mapping variable names to values.
	 * <p>Note: This attribute is not required to be supported by all
	 * HandlerMapping implementations. URL-based HandlerMappings will
	 * typically support it, but handlers should not necessarily expect
	 * this request attribute to be present in all scenarios.
	 */
```
We should find the other solution or use the old one as fallback in case of `webRequest.getAttribute(HandlerMapping.URI_TEMPLATE_VARIABLES_ATTRIBUTE, WebRequest.SCOPE_REQUEST)` null value.